pkg/generate/app/cmd/newapp_test.go

```diff
@@ -505,8 +506,10 @@ func TestBuildOutputCycleWithFollowingTag(t *testing.T) {
 	}
 	mockIS.Spec.Tags["10.0"] = image.TagReference{
 		From: &kapi.ObjectReference{
-			Kind: "DockerImage",
-			Name: "mockimage:latest",
+			Kind: "ImageStreamTag",
+			Name: "latest",
+			//Kind: "DockerImage",
+			//Name: "mockimage:latest",
```

Comment:
needs a test where input and output are in two different imagestreams (but input follows output).

Reply:
ok looking into it